### PR TITLE
updating to latest video token

### PR DESCRIPTION
**Pull request by vassjozsef**

### CODE REVIEW

File: rest/access-tokens/video-example/video-example.7.x.java [hunk 1]
```diff
@@ -10,12 +10,11 @@ public static void main(String[] args) {
     String twilioApiSecret = "xxxxxxxxxxxxxx";
 
     // Required for Video
-    String configurationProfileSid = "VSxxxxxxxxxxxx";
     String identity = "user";
       
     // Create Video grant
     VideoGrant grant = new VideoGrant();
-    grant.setConfigurationProfileSid(configurationProfileSid);
+    grant.setRoom('cool room');
```
Comment (joliveros) [inline rest/access-tokens/video-example/video-example.7.x.java]: For consistency you might want to use double quotes.

File: rest/access-tokens/video-example/video-example.5.x.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 $twilioApiSecret = 'xxxxxxxxxxxxxx';
 
 // Required for Video grant
-$configurationProfileSid = 'VSxxxxxxxxxxxx';
+$roomName = 'cool room';
 // An identifier for your app - can be anything you'd like
 $identity = "john_doe";
```
Comment (joliveros) [inline rest/access-tokens/video-example/video-example.5.x.php]: consistency with quotes/doublequotes

File: rest/access-tokens/video-example/video-example.7.x.java (hunk 1 above)
Comment (vassjozsef) [inline rest/access-tokens/video-example/video-example.7.x.java]: This is a bug and fixed.

File: rest/access-tokens/video-example/video-example.5.x.php (hunk 2 above)
Comment (vassjozsef) [inline rest/access-tokens/video-example/video-example.5.x.php]: Changed all to single quotes.

**Pull request merged**